testutil/retry/retry.go

```diff
@@ -0,0 +1,92 @@
+// Package retry provides support for repeating operations in tests.
+//
+// A sample retry operation looks like this:
+//
+//   func TestX(t *testing.T) {
+//       for r := retry.OneSec(); r.Next(t.FailNow); {
```

Comment:
Yeah, but `r.Next(t)` doesn't say well what it does with `t`. Maybe there is a better name for `Next`